.github/workflows/ci.yml

```diff
@@ -76,3 +76,19 @@ jobs:
         run: make VERBOSE=1 check
       - name: distcheck
         run: make VERBOSE=1 distcheck
+
+  test-redhat:
+    runs-on: ubuntu-latest
+    container: registry.access.redhat.com/ubi7/ubi
```

Comment:
Is there a way to add a comment linking to a description? The hex chars in https://catalog.redhat.com/software/containers/ubi7/ubi/5c3592dcd70cc534b3a37814 make it look like it's not meant to be a permanent link.